tests/vmi_lifecycle_test.go

```diff
@@ -996,6 +1005,65 @@ var _ = Describe("[rfe_id:273][crit:high][arm64][vendor:cnv-qe@redhat.com][level
 					fmt.Sprintf("Expected CPU model to equal to the default (%v)", v1.DefaultCPUModel),
 				)
 			})
+			It("[sig-compute]should add node selector to virt-launcher when setting default cpuModel in kubevirtCR", func() {
+				if len(supportedCpuModels) < 1 {
+					Skip("Must have at least one supported cpuModel for this test")
+				}
+				defaultCPUModel := supportedCpuModels[0]
+				config := originalConfig.DeepCopy()
+				config.CPUModel = defaultCPUModel
+				tests.UpdateKubeVirtConfigValueAndWait(*config)
+
+				newVMI := newCirrosVMI()
+				newVMI, err = virtClient.VirtualMachineInstance(util.NamespaceTestDefault).Create(newVMI)
```

Comment:
You can use `RunVMIAndExpectLaunch` instead of `Create` & `WaitFor...`

Reply:
Thanks for reviewing,
Done.